### PR TITLE
Containerized

### PR DESCRIPTION
**Pull request by bilalbaqar**
Signed-off-by: Bilal Baqar <mbilalbaqar@gmail.com>

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# enables logout from gdc login and nih login
+export REACT_APP_LOGOUT_NEXT=https://login.gdc.cancer.gov/logout
+
+# comma separated list of valid redirect uris
+export REACT_APP_LOGIN_REDIRECT_NOT_REQUIRED=https://localhost:3501,https://localhost:3000
+
+# debug namespace, e.g. "app". See https://www.npmjs.com/package/debug
+export REACT_APP_DEBUG=app
+export NODE_ENV=development
+export REACT_APP_TARGET_IS_DEMO=true
```
[the remark this answers is not in the document]
Comment (anncatton) [reply build.sh]: Oh yeah, the demo setting was to hide certain ui features that we hadn't decided whether we were going to include (and weren't entirely finished). I'm fairly certain those features won't ever be used in admin, but I'd have to confirm with Rosi.

[the remark this answers is not in the document]
Comment (anncatton) [reply build.sh]: For now if it's ok to leave set to true that would be good, I'll have to remove some things under that flag if we definitely aren't using them.

[the remark this answers is not in the document]
Comment (singergr) [reply build.sh]: Building it the same way we've been building it (demo=`true`) is good with me. :+1:

[the remark this answers is not in the document]
Comment (anncatton) [reply build.sh]: Great, thanks!

**Pull request merged**